### PR TITLE
{tools}[GCCcore/12.3.0] enchant-2 v2.6.5, hunspell v1.7.2, ITSTool v2.0.7, ...

### PR DESCRIPTION
**Pull request by jfgrimm**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jfgrimm** [comment]
@boegelbot: please test @ generoso

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
gpu07.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, 2 x NVIDIA NVIDIA A40, 535.86.10, Python 3.6.8
See https://gist.github.com/jfgrimm/4a6bbd99eabc0a17b52a0c9ab5ab4c9b for a full test report.

**Comment by jfgrimm** [comment]
@boegelbot: please test @ jsc-zen3

File: easybuild/easyconfigs/h/hunspell/hunspell-1.7.2-GCCcore-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'hunspell'
+version = '1.7.2'
+
+homepage = 'https://hunspell.github.io/'
+description = """Hunspell is a spell checker and morphological analyzer
+library and program designed for languageswith rich morphology and
```
Comment (verdurin) [inline easybuild/easyconfigs/h/hunspell/hunspell-1.7.2-GCCcore-12.3.0.eb:11]:
```suggestion
library and program designed for languages with rich morphology and
```

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/h/hunspell/hunspell-1.7.2-GCCcore-12.3.0.eb (hunk 1 above)
Comment (jfgrimm) [reply easybuild/easyconfigs/h/hunspell/hunspell-1.7.2-GCCcore-12.3.0.eb]: fixed

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 7 out of 7 (5 easyconfigs in total)
bear-pg0105u03b - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/e4d6275fcf3cea7ab4e896275bad8e5d for a full test report.

**Pull request merged**